core/trino-main/src/test/java/io/trino/sql/query/TestFilterInaccessibleColumns.java

```diff
@@ -164,19 +164,34 @@ public void testRowFilterWithUserOnNotAccessibleColumn()
     {
         accessControl.rowFilter(new QualifiedObjectName(TEST_CATALOG_NAME, TINY_SCHEMA_NAME, "nation"),
                 USER,
-                new ViewExpression(USER, Optional.of(TEST_CATALOG_NAME), Optional.of(TINY_SCHEMA_NAME), "comment IS NOT null"));
+                new ViewExpression(Optional.of(USER), Optional.of(TEST_CATALOG_NAME), Optional.of(TINY_SCHEMA_NAME), "comment IS NOT null"));
         accessControl.deny(privilege(USER, "nation.comment", SELECT_COLUMN));
         assertThatThrownBy(() -> assertions.query("SELECT * FROM nation WHERE name = 'FRANCE'"))
                 .hasMessage("Access Denied: Cannot select from columns [nationkey, regionkey, name, comment] in table or view test-catalog.tiny.nation");
     }
 
+    @Test
+    public void testRowFilterAsInvokerOnNotAccessibleColumn()
```

Comment:
NotAccessible -> Inaccessible

"RowFilterAsInvoker" -- what does this mean?
do we have RUN AS INVOKER and RUN AS DEFINER row filers?

Reply:
> Not at the moment, but we're trying to introduce that, basically.

Can we make the PR title convey that?

At first this looked like an optimization -- if filter owner is dumbly filled with session user, skip re-getting groups.

Reply:
> Can we make the PR title convey that?

I was under impression it already does convey that.  See `Allow ViewExpression use session user explicitly` where using session user is basically what SECURITY INVOKER is for views.

Reply:
You're right. It seems some things are obvious to you while they are not so obvious to me.